src/commands/latency-history.json

```diff
@@ -1,7 +1,7 @@
 {
     "HISTORY": {
         "summary": "Return timestamp-latency samples for the event.",
-        "complexity": "PATCH__TBD__40__",
+        "complexity": "O(1)",
```

Comment:
isn't that O(N) where N is the number of latency events recorded?
maybe GRAPH is too?

Reply:
@itamarhaber  said that the number of events is capped by some constant

Reply:
`#define LATENCY_TS_LEN 160`